src/DotNet.Status.Web/DotNet.Status.Web.csproj

```diff
@@ -27,7 +28,6 @@
     <PackageReference Include="Microsoft.Extensions.Configuration.AzureKeyVault" />
     <PackageReference Include="Microsoft.Extensions.Http" />
     <PackageReference Include="Octokit" />
-    <PackageReference Include="System.Text.Encodings.Web" />
```

Comment:
Won't this cause the warning to come back?

Reply:
Azure.Data.Tables requires a later version of System.Text.Encodings.Web, which floats everything up. So this specific upgrade isn't needed anymore.

Reply:
Oh, sneaky (and terrifying, that they are pulling up a framework dependency version for us)